doc/user_guide/security.rst

```diff
@@ -0,0 +1,127 @@
+Security
```

Comment:
Should we also tackle https://github.com/exasol/pyexasol/issues/174 or is this obsolete/duplicated?